<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OggSettings</class>
 <widget class="QWidget" name="OggSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="bitrateSetup">
     <property name="title">
      <string>Setup bitrate type and value</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="selectQuality">
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="vbrLayout">
        <item>
         <widget class="QLabel" name="minQuality">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Min</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="qualityValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maxQuality">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="selectBitrate">
        <property name="text">
         <string>Average bitrate, kBps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="abrLayout">
        <item>
         <widget class="QSlider" name="bitrateValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>48</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitrateValuePrecise">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>48</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectBitrate</sender>
   <signal>toggled(bool)</signal>
   <receiver>bitrateValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectQuality</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualityValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectBitrate</sender>
   <signal>toggled(bool)</signal>
   <receiver>bitrateValuePrecise</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrateValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrateValuePrecise</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrateValuePrecise</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrateValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
